mattermost-plugin/server/plugin_test.go

```diff
@@ -25,11 +31,18 @@ func TestServeHTTP(t *testing.T) {
 	assert.Nil(err)
 	bodyString := string(bodyBytes)
 
-	assert.Equal("Hello, world!", bodyString)
+	// TODO: fix this; where is `Hello World` text generated?
+	// assert.Equal("Hello, world!", bodyString)
```

Comment:
@jespino I left this as a TODO as I'm not sure of the intention.  This code comes from a very old PR https://github.com/mattermost/focalboard/pull/335.  I'm not seeing where the `Hello World` text is generated now.

Reply:
It was originally the code that was contained in the `/` of the plugin http server, I guess this is not needed.

See it here: https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/server/plugin.go#L25

Probably this tests just come with the starter template, see it here:

https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/server/plugin_test.go#L27

We can use any other endpoint to verify that this is working and returning the expected that. or just keep the test as is without the TODO.

Reply:
@jespino I added a simple `/hello` endpoint to act as a web service up check.  The unit test now uses that.